corpysprofiling/corpysprofiling.py

```diff
@@ -90,21 +92,22 @@ def corpus_analysis(corpus):
     """
     return
 
-def corpus_viz(corpus, stats):
+def corpus_viz(corpus, display=True):
```

Comment:
Suggest removing `display` argument, default should not print to screen.  
`def corpus_viz(corpus)`